lib/ohai/plugins/aix/uptime.rb

```diff
@@ -40,9 +40,9 @@
     when /^\d+-\d/
       (d, h, m, s) = so.split(/[-:]/)
     when /^\d+:\d+:\d/
-      (h, m, s) = so.split(/[:]/)
+      (h, m, s) = so.split(/:/)
     else
-      (m, s) = so.split(/[:]/)
+      (m, s) = so.split(/:/)
```

Comment:
FWIW, these could be the string `":"` instead.

Reply:
I saw that one and I'm going to see if there's a cop for it already and if not I'm going to get an issue created for that.